contracts/metagov/VeBalHelper.sol

```diff
@@ -83,23 +88,43 @@ contract VeBalHelper is Ownable {
         pcvDeposit.voteForGaugeWeight(token, gaugeWeight);
     }
 
+    /// @notice Stake tokens in a gauge
+    function stakeInGauge(address token, uint256 amount) external onlyOwner {
```

Comment:
These new functions should not really be on this contract. I'm guessing you've added them here because you're thinking of creating a boost and then delegating to this contract? 

That's a bad design imo - you should be delegating the boost to an EOA/multisig and not this fixed contract. What happens if there's another method that you might need to call in the future that is not accounted for here?

Reply:
Its just in the case we wanna stake BPTs from BalancerGaugeStaker contract. The boost would be delegated to that contract as well.

Reply:
Okay, fair enough. Will add tests for the new functions